packages/opentelemetry-api/src/metrics/BoundInstrument.ts

```diff
@@ -43,3 +44,12 @@ export interface BoundMeasure {
     spanContext: SpanContext
   ): void;
 }
+
+/** An Instrument for Observer Metric. */
+export interface BoundObserver {
+  /**
+   * Sets callback for observer
+   * @param callback
```

Comment:
I think it is worth to highlight that  callback gets executed (or invoked) every time before exporting this metric.

Reply:
but it is not executed every time - it execute this once, then sets observer for values and only observers are being called every time